portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,19 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.gallery-images {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: left;
+}
+
+.gallery-images img {
+    margin: 5px;
+    max-height: 400px;
+    max-width: 400;
+}
+
+.gallery-imagesimg img:hover {
```

Comment:
missing space between gallery-images and img? I don't seen a border on hover in the gallery.